akka-sample-sharding-java/build.sbt

```diff
@@ -0,0 +1,23 @@
+val akkaVersion = "2.5.3"
+
+val `akka-sample-sharding-java` = project
+  .in(file("."))
+  .settings(
+    organization := "com.typesafe.akka.samples",
+    scalaVersion := "2.12.2",
+    scalacOptions in Compile ++= Seq("-deprecation", "-feature", "-unchecked", "-Xlog-reflective-calls", "-Xlint"),
+    javacOptions in Compile ++= Seq("-Xlint:unchecked", "-Xlint:deprecation"),
+    javacOptions in doc in Compile := Seq("-Xdoclint:none"),
+    javaOptions in run ++= Seq("-Xms128m", "-Xmx1024m", "-Djava.library.path=./target/native"),
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-actor" % akkaVersion,
+      "com.typesafe.akka" %% "akka-remote" % akkaVersion,
+      "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
+      "com.typesafe.akka" %% "akka-cluster-sharding" % akkaVersion,
```

Comment:
not really needed to list all dependencies, `akka-cluster-sharding` would be enough here